.circleci/config.yml

```diff
@@ -1,7 +1,321 @@
+              sLnGj7NEeudwvKlyxuzj5CMmz111wVEI2olgQa9Sl+EBu140mnDNTNYCA7OnwE3z
+              GoFMOrXC2mf2ZfSge4orbL5Nellnt51pOLp2x8ECggEBALM8Mazw/FOF9mbdgjJM
+              PFGSaa7rBcVJwdHttDHBmlPI6wzsvFEMPru6nfx76KJQbORqK9r13sN5fyzof59m
+              TwsbMt/cFSnOQJ39M7YPstDofbl20cDOduUzpEVsRvVKokhqGB3XVRiuZ1y+8WSz
+              Wh7OiTu3AwzKsrcYXkZQdnlRBq0iYcfLPKzHqUJLLzbOH9Q6djL5c8V/qLNfvNI1
+              2HqKVqV8Ex+zKJhBWRAe+x3bKnbS7MPQ6zNfsOdgCmhydwRCquPzpr7JU/PFZh+4
+              b31cHgFrIZR2d2AzW1XcSLzsqa2vUs2RKOIu2deAPaUI/66zCZeTnGBNEFza76Ga
+              1oUCggEAA38oXcnputwL103SeD8+uwHjtTf183Rucr+Ryqz6GymiWjlzELqu7TRd
+              yadAaNg9CuXmYS33Jtk/UNS0k9FvYqGTR+SBXIZr6nt9ZFd0SNlQkwkAQCsuekEs
+              nJlmUZax7DxXMgIHMKDboHZYM/MhgzEGSALmhU5LZ76MS17v3NEPxYpVHxjAotxW
+              g03HjWTltS8Bgt6u0KFTGJKEUcfwvWKZtjk5Fc1heZ49zh1nU3zo9C/h8iiijTy2
+              s/YksP6cxveae4b7soN4rD/vnfsmKcG+DnTf6B8Zbm6tI2TneYOfFSCryp+yDnaJ
+              PIDNiTxNecePOmrD+1ivAEXcoL+e1w==
+              -----END PRIVATE KEY-----
+    authorizations:
+        - id: alice
+          key: fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu
+          signers:
+              - webextensions-rsa
+        - id: bob
+          key: 9vh6bhlc10y63ow2k4zke7k0c3l9hpr8mo96p92jmbfqngs9e7d
+          signers:
+              - webextensions-rsa-with-recommendation
+    ###########################################################################
+    #
+    #               The autograph configuration ends here.
+    #
+    ###########################################################################
+
+  defaults: &defaults
+    working_directory: ~/addons-server
+    docker:
+      # This is the python version we run in production.
+      - image: circleci/python:3.8
+      # Below are services this project depends on. In addition to these
+      # services, we also need autograph, which is started in the `test` job
+      # because we need to pass a configuration file to it and it's not
+      # possible in this section.
+      #
+      # Most settings below should be kept in sync with `docker-compose.yml`.
+      - image: redis:2.8
+      - image: memcached:1.4
+      - image: mysql:8.0
+        # This is needed to avoid an error with an authentication plugin.
+        command: --default-authentication-plugin=mysql_native_password
+        environment:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: olympia
+      - image: docker.elastic.co/elasticsearch/elasticsearch:6.8.8
+        environment:
+          # Disable all xpack related features to avoid unrelated logging in
+          # docker logs. https://github.com/mozilla/addons-server/issues/8887
+          xpack.security.enabled: false
+          xpack.monitoring.enabled: false
+          xpack.graph.enabled: false
+          xpack.watcher.enabled: false
+          discovery.type: single-node
+          cluster.name: default-cluster
+          ES_JAVA_OPTS: -Xms256m -Xmx256m
```

Comment:
Travis had a different set of options but I guess being closer to our local env setup is better, as long as we have enough resources on the host.